### PR TITLE
CATROID-625 Timer / Reset timer brick

### PR DESCRIPTION
**Pull request by Rawbat**
https://jira.catrob.at/browse/CATROID-625

Added sensor which tracks time since the start of the stage and a brick which resets the timer when executed. Also included a language test for these features.

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by blindguardian50 — APPROVED** [review]
Looks good to me!

File: catroid/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1079,6 +1079,9 @@
     <string name="brick_default_embroidery_file">embroidery.dst</string>
     <!--  -->
 
+    <!-- Device bricks -->
+    <string name="brick_reset_timer">Reset Timer</string>
```
Comment (wslany) [inline catroid/src/main/res/values/strings.xml:1083]:
```suggestion
    <string name="brick_reset_timer">Reset timer</string>
```
All our bricks use normal English capitalization rules. Also, see the jira ticket.

**Review by wslany — CHANGES_REQUESTED** [review]
Please change the text on the brick as indicated in the inline suggestion. 

Also, I just tried to import https://github.com/Catrobat/Catroid/raw/14c3887fba8e304bce93b33ad8b305367dba3ae0/catroid/src/androidTest/assets/catrobatTests/bricks/device/ResetTimer/testResetTimer.catrobat using https://jenkins.catrob.at/job/Catroid/job/PR-3768/lastSuccessfulBuild/artifact/catroid/build/outputs/apk/catroid/debug/catroid-catroid-debug-PR-3768-6.apk but there was an error and the project file could not be loaded.

**Review by christophsiegl — APPROVED** [review]
Looks good!

**Review by wslany — APPROVED** [review]

**Comment by wslany** [comment]
Thanks!

**Pull request merged**